Update links to non-existent EventListener article (#16216)
mdn/content: files/en-us/web/api/eventtarget/removeeventlistener/index.md

- `type`
  - : A string which specifies the type of event for which to remove an event listener.
- `listener`
  - : The [event listener](/en-US/docs/Web/API/EventTarget/addEventListener#the_event_listener_callback) function of the event handler to remove from the
    event target.
- `options` {{optional_inline}}
  - : An options object that specifies characteristics about the event listener.

    The available options are:

>     - `capture`:  A boolean value that specifies whether the [event listener](/en-US/docs/Web/API/EventTarget/addEventListener#the_event_listener_callback) to be removed is registered as a capturing listener or not. If this parameter is absent, a default value of `false` is assumed.

Reviewer: ```suggestion
    - `capture`:  A boolean value that specifies whether the [event listener](/en-US/docs/Web/API/EventTarget/addEventListener#the_event_listener_callback) to be removed is registered as a capturing listener or not. If this parameter is absent, a default value `false` is assumed.
```

- `useCapture` {{optional_inline}}
  - : A boolean value that specifies whether the [event listener](/en-US/docs/Web/API/EventTarget/addEventListener#the_event_listener_callback) to be removed is registered as a

>     capturing listener or not. If this parameter is absent, a default value of `false` is assumed.

Reviewer: ```suggestion
    capturing listener or not. If this parameter is absent, the default value `false` is assumed.
```
